WooCommerce/Classes/ViewRelated/Products/Edit Product/Subscription/SubscriptionPeriod+Limit.swift

```diff
@@ -0,0 +1,18 @@
+import enum Yosemite.SubscriptionPeriod
+
+extension SubscriptionPeriod {
+    /// Returns the max allowed limit for the period
+    ///
+    var limit: Int {
```

Comment:
Nit: This should be renamed to `freeTrialLimit` because the logic is specific to free trial period only.

```suggestion
    var freeTrialLimit: Int {
```

Reply:
Done in 31f54dee3cd45473788aa0fcacf240963bafc6e8